packages/web-components/fast-foundation/.eslintrc.json

```diff
@@ -1,6 +1,13 @@
 {
     "extends": ["@microsoft/eslint-config-fast-dna", "prettier"],
     "rules": {
+        "import/extensions": [
```

Comment:
Is this to enforce extensions in TypeScript src files? If so I don't think it's necessary. ModuleResolution Node16 will throw compile errors if extensions don't exist.

Reply:
This was mistakenly added to this PR - it was needed during the playwright test file conversions.